### DISCUSSION TITLE
stream shut down after 10-15 minutes

### DISCUSSION THREAD

**Discussion by f520k510** [GerryDazoo/Slinger#548] [General]
I have 2 slingbox 350s on the same lan as my pc running slingbox_server.exe 4.0
One 350 is working fine while the other gets disconnected after 10-15 minutes of viewing.
I would appreciate any advice.
Thanks.

Here are the logs:
[Problem with 350_13 stream20230224.txt](https://github.com/GerryDazoo/Slinger/files/10832866/Problem.with.350_13.stream20230224.txt)

**Comment by GerryDazoo** [comment]
Not much info here. Are they both running the same version of firmware? Are you using the same viewer for each test? Are you watching the same content? You didn't post the config info, identical? Might be a LAN issue.  Might be power supply problem. Swap power supplies and see what happens.
Please rerun the test using SlingerPlayer.  https://github.com/GerryDazoo/SlingerPlayer

**Reply by f520k510** [reply]
Thanks for your reply, Gerry.  
I noticed if we run the server and the player on different pcs then the stream works fine.   
For now I will do it this way.

**Reply by f520k510** [reply]
The config.ini is identical btw.

**Reply by GerryDazoo** [reply]
This is on a MAC correct? Just for grins, when you connect to the server running on the same host try 127.0.0.1 as the IP address.

**Comment by f520k510** [comment]
It is on a mini pc running windows 7.
I used 127.0.0.1.

**Reply by GerryDazoo** [reply]
would you mind giving Slinger Player a try.
https://github.com/GerryDazoo/SlingerPlayer
